index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<title>Emelio's Site</title>
+<link rel="stylesheet" href="style.css"/>
+
+
+<header>
+  <h1 id="header"> Emilio Comiling</h1>
+  <img class= "image" src="emelio.jpg">
+  <h2>About Emelio </h2>
```

Comment:
Semantic HTML makes code easy to read for developers and machines, but also makes code more accessible to those with impairments/disabilities. There are semantic tags such as header, main, sections etc that will help screen readers. Research into Semantic HTML to get the full list of benefits and tags that can be used.